modules/kubernetes-addons/helm-addon/main.tf

```diff
@@ -43,6 +43,7 @@ resource "helm_release" "addon" {
     content {
       name  = each_item.value.name
       value = each_item.value.value
+      type  = try(each_item.value.type, null)
```

Comment:
this should be `string` by default... without it, many places fail

Reply:
this is an optional argument so `null` is what should be used to default to the resource's default value https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#type